#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
uint8 id0
uint8 id1
uint8 id2
uint8 option
uint8 major_version
uint8 minor_version